packages/sdk/src/dataProtector/sharing/subgraph/getProtectedDataById.ts

```diff
@@ -15,13 +14,32 @@ export async function getProtectedDataById({
         id: "${protectedDataAddress}"
       ) {
         id
+        name
         owner {
           id
         }
+        collection {
+          id
+          owner {
+            id
+          }
+        }
+        isRentable
+        isIncludedInSubscription
+        isForSale
```

Comment:
I decided to reuse this subgraph function and just to add the fields I needed. It's fine for current callers of this method, everything will still work, they will just get unnecessary fields but that's ok.